### PR TITLE
China dnx

### PR DESCRIPTION
**Pull request by cristinamanum**
The script and changes for the catlog2dnx for China.

### CODE REVIEW

**Comment by skofman1** [comment]
What will be the China suffix?

File: src/Ng/CommandHelpers.cs [hunk 1]
```diff
@@ -216,9 +219,12 @@ public static StorageFactory CreateSuffixedStorageFactory(string suffix, IDictio
                 var storageKeyValue = arguments.GetOrThrow<string>(argumentNameMap[Arguments.StorageKeyValue]);
                 var storageContainer = arguments.GetOrThrow<string>(argumentNameMap[Arguments.StorageContainer]);
                 var storagePath = arguments.GetOrDefault<string>(argumentNameMap[Arguments.StoragePath]);
+                var storageSuffix = arguments.GetOrDefault<string>(argumentNameMap[Arguments.StorageSuffix]);
 
                 var credentials = new StorageCredentials(storageAccountName, storageKeyValue);
-                var account = new CloudStorageAccount(credentials, true);
+                var account = storageSuffix == null ?
```
Comment (shishirx34) [inline src/Ng/CommandHelpers.cs]: string.IsNullOrEmtpy(storageSuffix) ?

File: src/Ng/Scripts/Catalog2DnxV3China.cmd [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+	echo "Starting job - #{Jobs.ngcatalog2dnxChina.Title}"
+
+	title #{Jobs.ngcatalog2dnxChina.Title}
+	
+    start /w Ng.exe catalog2dnx -source #{Jobs.ngcatalog2dnx.Catalog.Source} ^
+        -contentBaseAddress #{Jobs.ngcatalog2dnxChina.ContentBaseAddress} ^ 
+        -storageBaseAddress #{Jobs.ngcatalog2dnxChina.StorageBaseAddress} ^ 
+        -storageType azure ^ 
+        -storageAccountName #{Jobs.common.v3China.StorageAccountName} ^ 
+        -storageKeyValue #{Jobs.common.v3ChinaStorage.Key} ^
```
Comment (shishirx34) [inline src/Ng/Scripts/Catalog2DnxV3China.cmd]: can we use a standard prefix for the variable names? 

`Jobs.Common.China.V3.StorageAccountName`
`Jobs.Common.China.V3.StorageKey`
`Jobs.China.ngCatalog2dns.ContentBaseAddress`
and so on...

Comment (cristinamanum) [inline src/Ng/Scripts/Catalog2DnxV3China.cmd]: Updated

File: src/Ng/CommandHelpers.cs (hunk 1 above)
Comment (cristinamanum) [inline src/Ng/CommandHelpers.cs]: China suffix: core.chinacloudapi.cn
Updated  with string.IsNullOrEmpty(storageSuffix)

**Pull request merged**